roles/stage_items/tasks/stage_download.yml

```
@@ -6,6 +6,7 @@
     headers: "{{ stage_item.headers | default(omit) }}"
     username: "{{ stage_item.username | default(omit) }}"
     password: "{{ stage_item.password | default(omit) }}"
+    timeout: 30
```

```suggestion
    timeout: "{{ stage_item.timeout | default(stage_item_download_timeout) | default(omit) }}"
```

maybe something like this?